graphbolt/include/graphbolt/sampled_subgraph.h

```diff
@@ -0,0 +1,118 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/sampled_subgraph.h
+ * @brief Header file of sampled sub graph.
+ */
+
+#ifndef GRAPHBOLT_SAMPLED_SUBGRAPH_H_
+#define GRAPHBOLT_SAMPLED_SUBGRAPH_H_
+
+#include <torch/custom_class.h>
+#include <torch/torch.h>
+
+namespace graphbolt {
+namespace sampling {
+
+/**
+ * @brief Struct representing a sampled subgraph.
+ *
+ * Example usage:
+ *
+ * Suppose the subgraph has 3 nodes and 9 edges.
```

Comment:
below example has 9 edges or 4 edges?

Reply:
4 edges, see `indptr[-1]` and 'indices.size(0)'

Reply:
Got you, changed for fix